docs/news/posts/2024-10-18-exploring-aiida-database.md

```diff
@@ -0,0 +1,339 @@
+
+```bash
+$ wget 'https://archive.materialscloud.org/record/file?record_id=2396&filename=two_dimensional_database.aiida' -O two_dimensional_database.aiida
+```
+
+Next, we need to make the archive data available for the AiiDA environment, and there are two possibilities for this. The first and traditional way is to [set up an AiiDA profile](https://aiida.readthedocs.io/projects/aiida-core/en/stable/installation/guide_complete.html#create-a-profile) and import the archive data with
+
+```bash
+$ verdi archive import two_dimensional_database.aiida
+```
+
+The other way is to set up a read-only profile directly from an archive file, which is more suitable for our current use case, as we just want to investigate and analyze the contents without modifying or extending the data. In this case, we can use (Note: this was introduced in AiiDA v2.5.0 and thus is not available in the current Quantum Mobile):
+
+```bash
+$ verdi profile setup core.sqlite_zip --filepath two_dimensional_database.aiida
+```
+
+It's also important to note that this profile is running on the `core.sqlite_zip` storage backend, that can have [some limitations for some queries](https://aiida.readthedocs.io/projects/aiida-core/en/stable/topics/storage.html), but these don't affect the analysis of this blog post.
+
+
+## Finding band gap nodes
+
+As our goal is to filter materials based on their band gaps, the first step is to find at least one AiiDA node (i.e. its universally unique identifier (UUID)) that contains the band gap value. Based on this we can later see how it is organized relative to the structure node, which will help us write the AiiDA query to collect and filter based on all band gap values for all materials.
+
+### Using supplementary metadata
+
+When AiiDA databases are published, it is recommended that authors provide a supplementary metadata file to help users understand how data is internally linked together. For this current dataset, this is the case, as the authors have provided `structures.json` (inside the tar `2D_materials.tar.gz` from the [published entry](https://archive.materialscloud.org/record/2024.157)), which contains a list of all the computationally investigated structures, together with AiiDA UUID values for corresponding calculated properties. Here's an excerpt from that file:
+
+```json
+[
+    {
+        // ...
+        "relaxed_2D_structure_uuid": "ba8bbdd9-defc-4292-af7d-ed3eb73f778e",
+        "formula_reduced": "AgBr",
+        "band_gap": 1.3,
+        "bands_uuid": "7fbb633d-a224-46b7-84b1-74a3ceca81e0",
+        // ...
+    },
+    // ...
+]
+```
+where we can see for the silver bromide (AgBr) the AiiDA UUIDs for the relaxed structure itself and the bands property that we want to analyse (the band gap node is a close descendent of this node). This information is enough to continue to the next section, but let's also explore alternative ways to obtain it.
```

Comment:
```suggestion
where we can see for the silver bromide (AgBr) the AiiDA UUIDs for the relaxed structure itself and the bands property that we want to analyse (the band gap node is a close descendent of this node). This information is enough to continue to the next section, but let's also explore alternative ways to obtain it (we will come back to this at the end of this blog post, promised).
```

Reply:
I am not sure this is necessary. The alternative way of getting the UUID is explained in the next subsection, but i see how that might be confusing (basically what i meant with "Next section" is "Manually browsing the database"). Will update this to make it clearer.